content/en/security/cloud_security_management/setup/_index.md

```diff
@@ -69,28 +69,74 @@ CSM is available in three packages: [CSM Enterprise][1], [CSM Pro][2], and [CSM
 **Notes**: 
 
 - You can enable features that aren't included in your package at any time by following the instructions on the [CSM Setup page][4].
-- CSM Identity Risks and CSM Vulnerabilities are in beta. For setup instructions, see [Cloud Security Management Identity Risks][5] and [Setting up Cloud Security Management Vulnerabilities][6].
+For setup instructions, see [Cloud Security Management Identity Risks][5] and [Setting up Cloud Security Management Vulnerabilities][6].
```

Comment:
```suggestion
```

Reply:
@drichards-87 I can't tell what this is suggested to be changed to ?